exercises/practice/leap/.meta/config.json

```diff
@@ -19,5 +19,5 @@
     ]
   },
   "source": "JavaRanch Cattle Drive, exercise 3",
-  "source_url": "https://www.javaranch.com/leap.jsp"
+  "source_url": "http://www.javaranch.com/leap.jsp"
```

Comment:
If you notice, my note about this was that it redirects to a different URL and that we should change the URL to the one it redirects to: `https://coderanch.com/t/718816/Leap`

Reply:
Oh, I'm sorry. Correcting it...